### ISSUE TITLE
bring shopping list card 

### ISSUE THREAD

**Issue by sander363** [dotKrad/hass_bring_shopping_list_component#2]
stop working with the Home assistant Update 0.116.0

**Comment by carvan** [comment]
Yes I confirm that's the case. The entity is being listed in HA, but the card in the dashboard doesn't display anything even if within the entity attributes there's the information.
